index.bs

```diff
@@ -430,6 +432,13 @@ about current and past use of the API.
 Note: this does not imply keeping a log of the actual [=sensor readings=]
 which would have issues of its own.
 
+<h4 id="user-triggered-activation" dfn>User triggered activation</h4>
+
+The information whether the user agent is requested to access [=sensor readings=] due to
```

Comment:
I'd be a bit more specific here and tie this directly to the permission request in [Request Sensor Access](https://w3c.github.io/sensors/#request-sensor-access) and the [§5.1.4 Permission API](https://w3c.github.io/sensors/#permissions) section.